<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstPage</class>
 <widget class="QDialog" name="FirstPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="layout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userpw">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="reg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>没有账号？点击就送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="accept">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>忠诚！南下！</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="useradd">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="useraddpw1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="useraddpw2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="reg_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>注册！启动！</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>死去的记忆开始攻击我</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLineEdit" name="ip_edit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>IP地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="port_edit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>11451</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
